_data/authors.yml

```diff
@@ -31,3 +31,7 @@ tom_cooper:
     twitter: https://twitter.com/tomncooper
     github: tomncooper
     website: https://www.tomcooper.org.uk/
+maros_orsak:
+    name: Maros Orsak
+    github: see-quick
+    linkedin: https://www.linkedin.com/in/majkl-orsak/
```

Comment:
it's fine having it here but won't be reflected as a link in the author. Right now, only Twitter is supported nor GitHub and Linkedin. Just noticed that even @tomncooper added the same.

Reply:
Hmmm, can we somehow make it worked? :)

Reply:
@ppatierno